CONTRIBUTING.md

```diff
@@ -25,6 +25,8 @@ We manage our Python dependencies with [Pipenv], you can [install it via Pipsi]:
 
 ``` bash
 $ curl https://raw.githubusercontent.com/mitsuhiko/pipsi/master/get-pipsi.py | python3
+$ echo 'export PATH=/Users/{your_user_name}/.local/bin:$PATH' >> ~/.bashrc  # OPTIONAL, depending on your operating system and configuration
```

Comment:
I think people configure their `$PATH` in many different ways. It might be better just to note:

> Pipsi is installed in a location that you may have to add to your $PATH environment variable.
> You'll have to add $HOME/.local/bin to your $PATH in order to get access to the `pipsi` executable